features/page-breaks.yml.dist

```diff
@@ -56,10 +62,86 @@ compat_features:
   - css.properties.break-inside
   - css.properties.break-inside.paged_context
 
+  # baseline: high
+  # baseline_low_date: 2020-07-28
+  # baseline_high_date: 2023-01-28
+  # support:
+  #   chrome: ≤80
+  #   chrome_android: "80"
+  #   edge: ≤80
+  #   firefox: ≤72
+  #   firefox_android: "79"
+  #   safari: ≤13.1
+  #   safari_ios: ≤13.4
+  - css.properties.break-after.auto
+  - css.properties.break-after.avoid
+  - css.properties.break-after.left
+  - css.properties.break-after.page
+  - css.properties.break-after.right
+  - css.properties.break-before.auto
+  - css.properties.break-before.avoid
+  - css.properties.break-before.left
+  - css.properties.break-before.page
+  - css.properties.break-before.right
+  - css.properties.break-inside.auto
+  - css.properties.break-inside.avoid
+
+  # baseline: high
+  # baseline_low_date: 2021-09-07
+  # baseline_high_date: 2024-03-07
+  # support:
+  #   chrome: ≤80
+  #   chrome_android: "80"
+  #   edge: ≤80
+  #   firefox: "92"
+  #   firefox_android: "92"
+  #   safari: ≤13.1
+  #   safari_ios: ≤13.4
+  - css.properties.break-inside.avoid-page
+
+  # baseline: false
+  # support:
+  #   chrome: "50"
+  #   chrome_android: "50"
+  #   edge: "12"
+  #   safari: "10"
+  #   safari_ios: "10"
+  - css.properties.break-after.paged_context.avoid-page
+
+  # baseline: false
+  # support:
+  #   chrome: "51"
+  #   chrome_android: "51"
+  #   edge: "12"
+  #   firefox: "92"
+  #   firefox_android: "92"
+  - css.properties.break-inside.paged_context.avoid-page
+
+  # baseline: false
+  # support:
+  #   chrome: ≤80
+  #   chrome_android: "80"
+  #   edge: ≤80
+  #   safari: ≤13.1
+  #   safari_ios: ≤13.4
+  - css.properties.break-after.avoid-page
+  - css.properties.break-after.recto
+  - css.properties.break-after.verso
+  - css.properties.break-before.avoid-page
+  - css.properties.break-before.recto
+  - css.properties.break-before.verso
+
   # baseline: false
   # support:
   #   chrome: "1"
   #   chrome_android: "18"
   #   edge: "12"
   - css.properties.page-break-after.avoid
   - css.properties.page-break-before.avoid
+
+  # baseline: false
+  # support:
+  #   firefox: ≤72
+  #   firefox_android: "79"
+  - css.properties.break-after.always
+  - css.properties.break-before.always
```

Comment:
These two are not in the spec. Can you remove these here, and then open an issue on BCD flagging that they should not be marked as standard_track?

Reply:
Victor flagged elsewhere that these keys are in css-breaks-4, so I retract this comment.